alvr/server_core/src/connection.rs

```diff
@@ -723,14 +723,14 @@ fn connection_pipeline(
 
                     #[cfg(windows)]
                     if let Ok(id) = alvr_audio::get_windows_device_id(&device) {
-                        ctx.events_queue
-                            .lock()
-                            .push_back(ServerCoreEvent::SetOpenvrProperty {
+                        ctx.events_sender
```

Comment:
Maybe some of these should get an unwrap because missing them is kinda ugly. Missing a tracking frame doesn't matter but setting audio devices for example should always happen. Although it should never run into issues anyway except that I'm not sure if the lifetimes will be fine on shutdown (the sender should be destroyed first), so ignoring send errors might be better actually.

Reply:
Or just log a warning if it fails and then ignore it

Reply:
No, unwrap is bad. Imagine if the driver is shutting down and the poll thread has exited already. Since ServerCore and event receiver are separate objects, at the server_core level we don't know which one will be dropped first

Reply:
Then we'll just assume that it's robust enough to never have issues except when alvr is actually shutting down and just always ignore the result

Reply:
For shutting down there should be no problem. For startup there is no problem either because the receiver is created by the ServerCore constructor itself, events are just queued up inside the channel until they are read

Reply:
Yea that's what I meant, just worded it badly. The send should never error except on shutdown and then it's not actually an issue so we just always ignore the errors